fastparquet/parquet.thrift

```diff
@@ -312,6 +312,10 @@ enum Encoding {
   /** Dictionary encoding: the ids are encoded using the RLE encoding
    */
   RLE_DICTIONARY = 8;
+
+  /** Custom lossy rice encoding
+   */
+  LOSSY_RICE = 19850410;
```

Comment:
Don't add library specific encodings and Thrift values, this defeats the purpose of Apache Parquet which is a **portable** columnar format. Just do a PR to add it to the general format if the benefit is significant: https://github.com/apache/parquet-format . Then you should also get feedback from other library implementers if they would support it.